### DISCUSSION TITLE
Getting error when I run denon start

### DISCUSSION THREAD

**Discussion by mylastore** [Prolifode/deno_rest#63] [Q&A]
error: Uncaught (in promise) ReferenceError: window is not defined
  window[symbol].idSequence = 0;
  ^
    at setupSham (https://deno.land/x/superdeno@4.8.0/src/xhrSham.js:8:3)
    at https://deno.land/x/superdeno@4.8.0/src/xhrSham.js:12:1

    info: window global is not available in Deno 2.
    hint: Replace `window` with `globalThis`.

**Comment by vicky-gonsalves** [comment]
> error: Uncaught (in promise) ReferenceError: window is not defined
>   window[symbol].idSequence = 0;
>   ^
>     at setupSham (https://deno.land/x/superdeno@4.8.0/src/xhrSham.js:8:3)
>     at https://deno.land/x/superdeno@4.8.0/src/xhrSham.js:12:1
> 
>     info: window global is not available in Deno 2.
>     hint: Replace `window` with `globalThis`.
> 

This [PR](https://github.com/Prolifode/deno_rest/pull/66) solves the issue for deno 2.x compatibility
